draginolht65n/sensor_test.go

```diff
@@ -0,0 +1,36 @@
+package draginolht65n
```

Comment:
pretty much everything is already covered by `node_test.go`. Once we add more dragino specific fields we will probably add more to here.